aws/resource_aws_ssm_resource_data_sync.go

```diff
@@ -15,6 +15,13 @@ func resourceAwsSsmResourceDataSync() *schema.Resource {
 		Read:   resourceAwsSsmResourceDataSyncRead,
 		Delete: resourceAwsSsmResourceDataSyncDelete,
 
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				d.Set("name", d.Id())
```

Comment:
Why not reading the name line 103? as in: `d.Set("name", syncItem.SyncName)`

Reply:
You're right! Thank you!
```
TF_ACC=1 go test ./aws -v -run=TestAccAWSSsmResourceDataSync_import -timeout 120m
=== RUN   TestAccAWSSsmResourceDataSync_import
--- PASS: TestAccAWSSsmResourceDataSync_import (68.67s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	68.710s
```